nan_callbacks.h

```diff
@@ -101,7 +101,7 @@ static const int kIndexPropertyQueryIndex =      5;
 static const int kIndexPropertyFieldCount =      6;
 
 template<typename T, typename P>
-P *GetWrapper(T needle) {
+static inline P *GetWrapper(T needle) {
```

Comment:
Also, the behavior is well-defined on POSIX and Windows and that's all we care about.  The few people still running VMS or DOS can fend for themselves. :-)